sdks/nodejs/lib/sdk_grpc_pb.js

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Maybe don't need to call header() on this file in the build script as the boilerplate is already included?

Reply:
it seems so. This is good, we just need to include make targets !